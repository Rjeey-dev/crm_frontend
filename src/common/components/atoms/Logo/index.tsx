import React from 'react';

import Img from 'common/components/atoms/Img/index';
import {URL_HOME_LOCALIZED} from "common/routes/paths";
import LocalizedLink from "containers/links/LocalizedLink";

interface IProps {
    classes?:string,
}

function Logo(props: IProps) {
    return <LocalizedLink to={URL_HOME_LOCALIZED} className={props.classes}><Img src='/static/images/logo.png' alt='Logo' lazyLoad={false}/></LocalizedLink>;
}

export default Logo;