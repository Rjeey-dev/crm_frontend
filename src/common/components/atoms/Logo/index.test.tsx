import { shallow } from 'enzyme';
import React from 'react';

import Logo from './index';

it("Renders <Logo/> with correct properties", () => {
    const output = shallow(
        <Logo href='dashboard' classes='class' src='/images/logo.png'/>
    ).html();

    expect(output).toMatchSnapshot();
});

it("Renders <Logo/> with only required props", () => {
    const output = shallow(
        <Logo href='dashboard' src='/images/logo.png'/>
    ).html();

    expect(output).toMatchSnapshot();
});