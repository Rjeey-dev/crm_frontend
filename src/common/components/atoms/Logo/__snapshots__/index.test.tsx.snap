// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Logo/> with correct properties 1`] = `"<a href=\\"dashboard\\" class=\\"class\\"><img src=\\"/images/logo.png\\" alt=\\"Logo\\"/></a>"`;

exports[`Renders <Logo/> with only required props 1`] = `"<a href=\\"dashboard\\"><img src=\\"/images/logo.png\\" alt=\\"Logo\\"/></a>"`;
